### PR TITLE
Added guest article: Analysis of Covid 19 Data with pandas and matplotlib

### PR DESCRIPTION
**Pull request by jarviscodes**
Let me know if it needs any changes!

### CODE REVIEW

**Comment by clamytoe** [comment]
Although I couldn't see any of the images through the git app, the md file sounds great! You did some really nice sleuthing there. I really liked your use of Grouper.

The only suggestion would be to maybe drop in an explain for what a DataFrame is. You jumped straight into a df.head() and someone unfamiliar with Pandas might just get lost there. 

Nice job!

**Comment by jarviscodes** [comment]
@clamytoe Thanks for the feedback, you're right. I've added a bit more information about the pandas library and dataframes!

File: content/guest-covid-19-data-analysis-pandas.md [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+```
+
+And if these differ, we can download the data to our data folder! 
+
+Have a look at `datafetcher.py` on [my github](https://github.com/jarviscodes/covid-19-view/blob/master/code/datafetcher.py) to see how I pulled all this together.
+
+<a name="forest-trees"></a>
+### Part 2: The forest and the trees.
+
+In Dutch there's a saying:
```
Comment (bbelderbos) [inline content/guest-covid-19-data-analysis-pandas.md:75]: :)

**Comment by bbelderbos** [comment]
Interesting post! Thanks Cedric, pushed it (but GH outage so let's see when it appears)

**Pull request merged**